test/unit/transform/ferc1_test.py

```diff
@@ -330,3 +332,47 @@ def test_drop_duplicate_rows_dbf():
     )
     with pytest.raises(AssertionError):
         drop_duplicate_rows_dbf(df_null_w_unique_data, params=params)
+
+
+def test_unstack_balances_to_report_year_instant_xbrl():
+    """Test :meth:`unstack_balances_to_report_year_instant_xbrl`."""
```

Comment:
I wasn't sure how to structure the test functions. Some in here are multiple tests in one function (see `test_drop_duplicate_rows_dbf`) and some have multiple functions to tests different types of fails/passes (see `test_wide_to_tidy`, `test_wide_to_tidy_fail`, `test_wide_to_tidy_rename`)

Reply:
I think the one function per small test is more of a best practice because it easier to see from even just the test name what the failure actual is. But you are right we really haven't been doing that across the board in this module. I was reluctant to make global variables like your `df` that we then altered to test the assertions. Because altering global variables is generally a no-no. Even though I think altering copies would be okay.

These tests aren't super complex so I think either way here is fine.

Reply:
oh this is a baby nit but i think the doc here should be :func:`unstack_balances_to_report_year_instant_xbrl`